{"name": " 输入错误的账号或密码，登录失败", "status": "passed", "parameters": [{"name": "user", "value": "'123456'"}, {"name": "pwd", "value": "'123456'"}], "start": 1585811964829, "stop": 1585811965060, "uuid": "d9271e8b-5e09-40ec-b385-4676f33c0a7a", "historyId": "45338e2d532f764ec551752f153d1b58", "testCaseId": "abbc5fc359ed88434bc4c6d645012222", "fullName": "case.login_case.test_login#test_login_03", "labels": [{"name": "story", "value": "测试登录接口"}, {"name": "parentSuite", "value": "case.login_case"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "goushuai"}, {"name": "thread", "value": "15340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.login_case.test_login"}]}